src/main/java/org/apache/commons/io/input/Tailer.java

```diff
@@ -556,4 +784,181 @@ private long readLines(final RandomAccessFile reader) throws IOException {
             return rePos;
         }
     }
+
+    /**
+     * Abstraction on {@link java.io.File} which allows substitution of remote files for example using jCIFS
+     *
+     * @since 2.12.0
+     */
+    public interface Tailable {
+        /**
+         * @return  The name of the file denoted by this tailable
+         */
+        String getName();
```

Comment:
The name of... what? the base file name? The full file name?

Reply:
I've renamed this to `getFileName` more descriptive but now inconsistent with `java.io.File#getName`